champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/sword/WolfsPounce.java

```diff
@@ -171,40 +140,51 @@ public void loadSkillConfig() {
     }
 
     @Override
-    public void activate(Player player, int level) {
-        final ChargeData chargeData = new ChargeData(level);
-        charging.put(player, chargeData);
+    public void trackPlayer(Player player) {
+        // Action bar
+        Gamer gamer = championsManager.getClientManager().search().online(player).getGamer();
+        gamer.getActionBar().add(900, actionBarComponent);
     }
 
     @Override
-    public boolean shouldDisplayActionBar(Gamer gamer) {
-        return false;
+    public void invalidatePlayer(Player player) {
+        // Action bar
```

Comment:
I know this is unchanged, but wouldn't it be wiser if we just pass the gamer and player as arguments into invalidatePlayer and similar? I imagine this code to get the gamer is quite frequent?

If we only do it a couple of times don't worry about it

Reply:
We do it exactly 10 times in that same method, I suppose we can do that, yes.